### PR TITLE
feat(visibility): add info message for opt in users

### PR DESCRIPTION
**Pull request by Chrissy**
image

### CODE REVIEW

File: src/sentry/static/sentry/app/components/alertMessage.jsx [hunk 1]
```diff
@@ -27,11 +27,6 @@ const StyledCloseButton = styled.button`
   right: ${p => p.theme.grid}px;
   top: 7px;
 
-  /* stylelint-disable-next-line no-duplicate-selectors */
```
Comment (Chrissy) [inline src/sentry/static/sentry/app/components/alertMessage.jsx]: this inherits the text color in the svg which is better than always setting it to gray (which looks weird)

**Review by billyvg — APPROVED** [review]

**Comment by lynnagara** [comment]
Can we check the sentry10 feature flag? I think Discover and events users with this header should get this message

**Comment by Chrissy** [comment]
@lynnagara yeah I agree. Adding it now.

**Comment by Chrissy** [comment]
added!

**Review by markstory — APPROVED** [review]
Looks good. I added a test for the new alertstore behavior as I feel its something we'll want to use in the future.

**Comment by Chrissy** [comment]
test lgtm. once https://github.com/getsentry/sentry/pull/11965 goes through we can rebase and merge this.

**Comment by Chrissy** [comment]
ah. there it went!

File: src/sentry/static/sentry/app/components/organizations/globalSelectionHeader/globalSelectionHeader.jsx [hunk 2]
```diff
@@ -98,6 +99,20 @@ class GlobalSelectionHeader extends React.Component {
       return;
     }
 
+    const hasSentry10 = new Set(this.props.organization.features).has('sentry10');
```
Comment (EvanPurkhiser) [inline src/sentry/static/sentry/app/components/organizations/globalSelectionHeader/globalSelectionHeader.jsx]: We should talk about this in TCS as well @billyvg, we inconsistetly at times use `Set().has(...)` for features, other times just do `org.features.includes(...)`.

Comment (Chrissy) [reply src/sentry/static/sentry/app/components/organizations/globalSelectionHeader/globalSelectionHeader.jsx]: yeah i noticed this and was confused.  seems like the second is simpler?

**Pull request merged**